sat-plugin/src/main/resources/rulesets/pmd/rules.xml

```diff
@@ -53,10 +53,10 @@
     <priority>3</priority>
   </rule>
   <rule ref="rulesets/java/logging-java.xml/AvoidPrintStackTrace">
-    <priority>3</priority>
+    <priority>1</priority>
```

Comment:
Have you tested it on openHAB-addons and ESH builds? Are we sure that it will not break the build ?

Reply:
I created a PR for openhab, have to check for ESH

Reply:
Hmm, I have bigger problems when building openhab-core 

```
[ERROR] org.openhab.core.internal.icon.IconForwarder.java:[2]
Header line doesn't match pattern ^ \* Copyright \(c\) 2010-2018 by the respective copyright holders\.$
```

Reply:
Yes, this is due to the recent pull request from @tanyatanyatanya, that was merged yesterday... So we have to fix the repositories before using this SAT version on them.

Reply:
I think the reason for these problems is that the copyright headers in `openhab-core` start with 
> Copyright (c) 2015-2018

but the copyright headers in `openhab2-addons` and the expected header pattern in the `static-code-analysis` tool are
 > Copyright (c) 2010-2018

I will add a `ruleset.properties` file in openhab-core that overrides the expected copyright header pattern for the openhab-core repo and create a PR.

Reply:
Already opened, waiting for the new release : https://github.com/openhab/openhab-core/pull/262